src/views/Layout.jsx

```diff
@@ -21,15 +21,24 @@ export function Layout() {
 					<Outlet />
 				</main>
 				<nav className="Nav">
-					<a href="#" className="Nav-link">
+					{/* <a href="#" className="Nav-link">
 						Home
 					</a>
 					<a href="#" className="Nav-link">
 						List
 					</a>
 					<a href="#" className="Nav-link">
 						Add Item
-					</a>
+					</a> */}
+					<NavLink to="/" className="Nav-link">
+						Home
+					</NavLink>
+					<NavLink to="/list" className="Nav-link">
+						List
+					</NavLink>
+					<NavLink to="/add-item" className="Nav-link">
+						Add Item
+					</NavLink>
 				</nav>
```

Comment:
I think these `<NavLink>` components are part of the other task that the other team is working on. Looking at their PR it has this same code. There might be a merge conflict with this code. If I am not understanding something, please let me know.

https://github.com/the-collab-lab/tcl-48-smart-shopping-list/pull/16/files

Reply:
I updated our code